### ISSUE TITLE
Isolating grammar and style checking

### ISSUE THREAD

**Issue by glocalglocal** [writingtool-org/writingtool#30]
In trying to isolate Grammar and Style checking, I created a Grammar_only profile and unticked all style rules. When I look again, all style rules remain unticked except one which is always re-ticked: 'British English, Oxford spelling...' 

Ideally, switching between all, grammar-only and style-only should be easier with selectors, rather that we having to tick multiple items. There could me a master switch for all optional rules too. Or there could be predefined profiles that users can't change.

**Comment by FredKruse** [comment]
> 'British English, Oxford spelling...'
Under what category did you find this rule?

The other ideas are enhancements, that I put on the list for WT 1.2

**Comment by FredKruse** [comment]
I found the category and identified the bug. It is solved. Please test it with the nightly tomorrow and give me feedback.

**Comment by FredKruse** [comment]
The Buttons are installed. Please test the snapshot tomorrow and give me feedback.

**Comment by glocalglocal** [comment]
I don't see any new buttons in the version dated 3 January.

**Comment by FredKruse** [comment]
There was a problem with my infrastructure. So there were no new snapshots on the server.
I fixed the problem. The buttons appear in the last snapshot from last night.

**Comment by glocalglocal** [comment]
The version dated 6 January shows the buttons and they work but: 
Several button under Profiles (eg Add Profile..., Rename...) now generate this pop-up: 
![image](https://github.com/user-attachments/assets/9b60eba7-2d7a-473a-9ba7-de1f35bb73ce)

**Comment by FredKruse** [comment]
There is a problem with the FlatLight and FlatDark themes when using default java methods for displaying messages, information, choosing colors etc.
I have installed a workaround. These components are switched back to the standard style. It's not pretty, but it's the only thing that works so far. I will try to contact the developers of the themes.
Please test the version tomorrow and give me feedback.

**Comment by FredKruse** [comment]
Yes, that is the workaround. I opened an issue at FlatLaf in the hope that there is a solution for the message dialogs.

**Comment by FredKruse** [comment]
I am closing this issue. The remaining open point should be discussed under issue #29
